FastSurferCNN/data_loader/conform.py

```diff
@@ -232,17 +257,24 @@ def findMinSizeConformDim(img, max_size=1, min_dim=256):
     return min_size, conform_dim
 
 
-def conform(img, order=1, conform_min=False):
+def conform(img: nib.analyze.SpatialImage, order: int = 1, conform_min: bool = False, dtype: Optional[Type] = None) -> nib.MGHImage:
```

Comment:
why has dtype been added, when it is neither used in the code below (nor documented in the help text)?